mail_restrict_auto_follower/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import models
+
+
+class ResPartner(models.Model):
+    """This is for avoid inheritance problem:
+    https://github.com/odoo/odoo/issues/9084"""
+    _name = 'res.partner'
```

Comment:
I had the same problem, have a look at this solution: https://github.com/OCA/social/pull/28/files#diff-371a424a175e245428d526a9c14007bfR82